src/types/excelsheet.rs

```diff
@@ -352,17 +355,23 @@ impl ExcelSheet {
 
         let available_columns = sheet.get_available_columns();
 
+        let mut aliased_available_columns = Vec::with_capacity(available_columns.len());
+
+        available_columns.iter().for_each(|column_name| {
+            aliased_available_columns.push(alias_for_name(column_name, &aliased_available_columns))
```

Comment:
ah right yes!
why not
```rs
let aliased_available_columns = available_columns
    .iter()
    .map(|column_name| alias_for_name(column_name, &available_columns))
    .collect::<Vec<_>>()
``` 
?

Reply:
[2f1445](https://github.com/ToucanToco/fastexcel/pull/191/commits/42f1445c1c94b241eea7587a037db2a2349d96e9)